src/vs/workbench/browser/parts/editor/editorCommands.ts

```diff
@@ -1389,8 +1390,8 @@ function getEditorsContext(accessor: ServicesAccessor, resourceOrContext?: URI |
 	};
 }
 
-export function getEditorsFromContext(accessor: ServicesAccessor, resourceOrContext?: URI | IEditorCommandsContext, context?: IEditorCommandsContext): { editor: EditorInput; group: IEditorGroup }[] {
-	const { editors, groups } = getEditorsContext(accessor, resourceOrContext, context);
+export function resolveEditorsContext(context: EditorsContext): { editor: EditorInput; group: IEditorGroup }[] {
+	const { editors, groups } = context;
 
 	const editorsAndGroup = editors.map(e => {
 		if (e.editorIndex === undefined) {
```

Comment:
Looking at the type it can be undefined.

`getEditorsContext` is a function that gets the context of a command by taking multiselect into account. This function already existed.

This `resolveEditorContext` function was introduced to make it easier to resolve the context, the same way `resolveCommandsContext` does this.

I suggest looking into this entire file during debt week as it has been touched by many different people